WhateverGreen/kern_igfx.cpp

```diff
@@ -782,14 +783,21 @@ bool IGFX::applyPlatformInformationListPatch(uint32_t framebufferId, Framebuffer
 	return framebufferFound;
 }
 
+template <>
+void IGFX::applyPlatformInformationPatchEx(FramebufferHSW *frame) {
+	// fCursorMemorySize is Haswell specific
+	if (framebufferPatchFlags.bits.FPFFramebufferCursorSize) {
+		frame->fCursorMemorySize = fPatchCursorMemorySize;
+		DBGLOG("igfx", "fCursorMemorySize: 0x%08X", frame->fCursorMemorySize);
+	}
+}
+
 template <typename T>
 bool IGFX::applyPlatformInformationListPatch(uint32_t framebufferId, T *platformInformationList) {
 	auto frame = reinterpret_cast<T *>(findFramebufferId(framebufferId, reinterpret_cast<uint8_t *>(platformInformationList), PAGE_SIZE));
 	if (!frame)
 		return false;
 
-	bool r = false;
```

Comment:
I am afraid it is not the variable here being an error, but rather no use of it in the bottom where we have return true. Should probably be restored, relabled like in the other function, and the return condition should be fixed in the bottom.